OutboundCallReminder/OutboundFunction_AzurePortal/readme.md

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Please refer the comments in the above readme.md file and implement same here as well which ever applicable.